pkg/reconciler/contour/resources/kingress.go

```diff
@@ -31,6 +31,10 @@ import (
 	"knative.dev/pkg/logging"
 )
 
+// MakeEndpointProbeIngress creates a new child kingress resource with a
```

Comment:
```suggestion
// MakeEndpointProbeIngress creates a new child KIngress resource with a
```

Reply:
I prefer `kingress` because it matches the kubectl short name 🙃